my-contacts/src/Contact.treat.js

```diff
@@ -0,0 +1,100 @@
+import { style } from 'treat';
+
```

Comment:
(max-width: 750px) is repeated 3 times. It may be worthwhile extracting this into a constant